moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -50,7 +50,15 @@ const useColumns = ({
           />
         ),
         renderEditCell: (props) => (
-          <SubprojectLookupComponent {...props} data={data} />
+          <LookupAutocompleteComponent
+            {...props}
+            name="project"
+            options={data?.subprojectOptions}
+            autocompleteProps={{
+              getOptionLabel: (option) =>
+                `${option.project_id} - ${option.project_name_full}`,
+            }}
```

Comment:
last one where the props could be declared out of scope so as to provide a stable object reference to the component 👍